control-plane/agents/src/bin/core/node/service.rs

```diff
@@ -119,6 +119,34 @@ impl NodeOperations for Service {
         let node = self.drain(id, label).await?;
         Ok(node)
     }
+
+    /// Apply the label to node.
+    async fn label(&self, id: NodeId, label: String) -> Result<Node, ReplyError> {
+        let mut label_map = HashMap::new();
+        for pair in label.split(';') {
```

Comment:
Should be comma `,` separated?

Reply:
Its separated by `;` . This is what kubectl label command does